### ISSUE TITLE
wooden fillet knives crash game 1.20.1

### ISSUE THREAD

**Issue by s4met1mes** [TeamMetallurgy/Aquaculture#486]
Hey! I've been trying to play with the mod but every time i try to use a furnace it give me the error code -1 and says that the wooden fillet knives are missing. Ive attached the crash report below.

It doesnt matter what i put in the furnace, every item i drop in there will automatically crash the game

All my mods are up to date, I use no other plugins or anything

[crash-2024-12-04_07.11.41-client.txt](https://github.com/user-attachments/files/18003487/crash-2024-12-04_07.11.41-client.txt)

**Comment by GirafiStudios** [comment]
Unable to reproduce.

**Comment by GirafiStudios** [comment]
Are you saying this happen whenever  you put a wooden fillet knife in a furnace, or anything at all in a furnace?

**Comment by s4met1mes** [comment]
Anything at all, i tried putting coal in a furnace and it crashed the entire game. It just gives me the error message that the wooden fillet knife is missing

**Comment by GirafiStudios** [comment]
Very weird!

Are you able to reproduce this with just Aquaculture 2 installed?

**Comment by s4met1mes** [comment]
Nope! must be some mods clashing with each other... Weird that it gives that error code though. I'll try troubleshooting!

**Comment by GirafiStudios** [comment]
Did you ever figure out what was causing the issues?

**Comment by s4met1mes** [comment]
Nope! I troubleshot in a singleplayer world and nothing went wrong, so must be on the server side which i dont have access to

**Comment by GirafiStudios** [comment]
I'll conduct a test on a server at some point, when I get time.

Only tested in Singleplayer as well.

Just to make sure - is the server running any sort of Forge/Bukkit hybrid? Meaning that the server has mods as well as plugins?,

**Comment by GirafiStudios** [comment]
Completely forgot about this, sorry!

Just tested on a server now as well & was still not able to reproduce. I suspect the server is most likely running some sort of Bukkit/Forge hybrid, meaning it's a server that can have both plugins and mods. Tends to cause issues.
